src/dg/strong_dg.cpp

```diff
@@ -1295,7 +1434,9 @@ void DGStrong<dim,nstate,real,MeshType>::assemble_boundary_term_strong(
 
         // Evaluate physical convective flux
         std::array<dealii::Tensor<1,dim,real>,nstate> conv_phys_flux;
-        conv_phys_flux = this->pde_physics_double->convective_flux (soln_state);
+        if(!this->all_parameters->use_split_form && !this->all_parameters->use_curvilinear_split_form){
+            conv_phys_flux = this->pde_physics_double->convective_flux (soln_state);
+        }
```

Comment:
can you explain why this if statement is added now throughout this section? `conv_phys_flux` isn't needed when `use_split_form == true` and `use_curvilinear_split_form == false`?

Reply:
It evaluates ocnv_phys_flux only if it's conservative DG strong, otherwise, if split form or curvilinear split form, it solves the entropy conserving 2pt flux form. Ideally, in the future the EC two-point flux would be its own type and DG strong would only be conservative, but for now I left if statements to differentiate between the two.